### PR TITLE
Polish translation

### PR DESCRIPTION
**Pull request by Kucharskov**
Amazing work by M. Kucharskov and Mezurashii.
Regards!

### CODE REVIEW

File: app/src/main/res/values-pl/strings.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<!-- Translated with love by M. Kucharskov and Mezurashii -->
```
Comment (CzBiX) [inline app/src/main/res/values-pl/strings.xml]: Please use space instead of tab

Comment (CzBiX) [inline app/src/main/res/values-pl/strings.xml]: Please use space instead of tab, thanks.

**Review by CzBiX — CHANGES_REQUESTED** [review]

**Comment by Kucharskov** [comment]
Its more important than new launguage in app. As if you couldnt do it yourself.
Anyways - fixed. Regards!

**Comment by CzBiX** [comment]
It's the way of collaboration. I think you should know what's wrong as the author of this commit.
Modify your code without contact you may be bad manners, or maybe I'm too sensitive.

Anyway thanks your work.

**Pull request merged**